src/bundle/Resources/public/js/scripts/fieldType/base/base-rich-text.js

```diff
@@ -291,10 +293,36 @@
             return html.querySelectorAll(allowedSelectors.join(','));
         }
 
+        cleanZeroWidthCharacters(tag) {
+            tag.innerHTML = tag.innerHTML.replace(/[\u200B-\u200D\uFEFF\u00A0]/g, '');
+            return tag;
+        }
+
+        stripInlineFormatting(html) {
+            const extractables = ['[data-ezelement=ezcontent]'];
+            const removables = ['[data-ezelement=ezconfig]'];
+
+            html.querySelectorAll(extractables.join(','))
+                .forEach((node) => {
+                    node.replaceWith(node.innerText);
+                });
+
+            html.querySelectorAll(removables.join(','))
+                .forEach((node) => {
+                    node.remove();
```

Comment:
I believe if you remove some node from this html it will not be stored in textarea and not send to the backend.